### DISCUSSION TITLE
THE SIGNATURE IS ERASED ON TABLET AND ANDROID

### DISCUSSION THREAD

**Discussion by jesucr4** [szimek/signature_pad#663] [Q&A]
At the moment I make a signature and write in an input, when the keyboard appears the signature is deleted, what could I do?

https://user-images.githubusercontent.com/55533775/195847143-357197d1-4ca2-4e4d-9ab5-daa455b85a43.mp4

**Comment by UziTech** [comment]
The canvas is cleared by the browser automatically on resize so you can add the data back to it with:

```
signaturePad.fromData(signaturePad.toData())
```

**Reply by jesucr4** [reply]
that I should add it when I call the method resize()??

**Reply by UziTech** [reply]
Yes. You can see an example in our demo
https://github.com/szimek/signature_pad/blob/bf6c8777e036141b7425cba78e9bb6ca5cb66058/docs/js/app.js#L19
